templates/financial-services/resources.html

```diff
@@ -27,239 +31,309 @@ <h2>Canonical &ndash; Trusted source of open source for financial services</h2>
+          <span class="p-pull-quote__citation">Georgi Georgiev &mdash; CIO, SBI BITS</span>
+        </blockquote>
+      </p>
     </div>
-  </article>
-
-  <hr class="u-fixed-width is-muted">
-
-  <article class="p-strip is-shallow">
-    <div class="row">
-      <div class="col-4">
-        <h3 class="p-heading--4"><a href="/engage/k8s-allan-gray">Allan Gray unlocks unprecedented availability and productivity with Charmed Kubernetes</a></h3>
-      </div>
-      <div class="col-4">
-        <p>Allan Gray, an investment banking company, originally managed their Kubernetes cluster in-house, but soon realised lack of additional support and the resourcing associated with its management could pose issues.</p>
-        <p>To minimise risk and maximise revenues, Allan Gray turned to Canonical's Charmed Kubernetes to boost efficiency and availability.</p>
-      </div>
-      <div class="col-4 u-vertically-center u-align--center u-hide--small u-hide--medium">
+    <div class="col-4 p-card">
+      <a href="/engage/k8s-allan-grayengage/sbi-casestudy" class="no-border u-align--center u-vertically-center" style="min-height: 7rem;">
         {{ image (
           url="https://assets.ubuntu.com/v1/0b5c73a8-allan-gray-logo.png",
           alt="",
-          width="250",
-          height="48",
+          width="200",
+          height="38",
           hi_def=True,
           loading="lazy"
           ) | safe
         }}
-      </div>
+      </a>
+      <hr style="margin-bottom: 1rem;" />
+      <h3 class="p-heading--4">
+        <a href="/engage/k8s-allan-gray">Allan Gray unlocks unprecedented productivity with Charmed Kubernetes</a>
+      </h3>
+      <p class="p-card__content">Allan Gray, an investment banking company, managed their Kubernetes cluster in-house, but soon realised lack of additional support and management resourcing could pose issues. To minimise risk and maximise revenues, it turned to Canonical's Charmed Kubernetes.</p>
     </div>
-  </article>
+  </div>
 </section>
 
-
 <section class="p-strip--light">
   <div class="u-fixed-width">
     <h2>Whitepapers</h2>
   </div>
-
   <article class="p-strip is-shallow">
     <div class="row">
-      <div class="col-5">
+      <div class="col-4">
         <h3 class="p-heading--4"><a href="/engage/kubernetes-and-containerisation-for-financial-services">Kubernetes and Containerisation for Financial Services</a></h3>
         <p class="p-heading--5">A CIO's guide to unlocking success in digital transformations</p>
       </div>
-      <div class="col-7">
+      <div class="col-5">
         <p>Containerisation and Kubernetes have become inextricably woven into the digital transformation imperative. This white paper looks at the role of containerisation and Kubernetes in the cloud native transformation of financial services. By understanding why these technologies are so important and how best to leverage them, financial institutions can execute their digital strategy faster.</p>
```

Comment:
This paragraph has been removed in the copy doc

Reply:
Just checked with Anastasia, we decided to keep them...! I'll update the copy docs.